projects/components/src/stepper/stepper.module.ts

```diff
@@ -0,0 +1,16 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { MatStepperModule } from '@angular/material/stepper';
+import { MemoizeModule } from '@hypertrace/common';
+import { ButtonModule } from '../button/button.module';
+import { IconModule } from '../icon/icon.module';
+import { LabelModule } from '../label/label.module';
+import { StepperTabComponent } from './stepper-tab.component';
+import { StepperComponent } from './stepper.component';
+
+@NgModule({
+  imports: [CommonModule, MatStepperModule, ButtonModule, IconModule, LabelModule, MemoizeModule],
```

Comment:
nit: Sometimes is easier to read if you keep an alphabetic order with the imports.

Reply:
updated